numpy-mp/Dockerfile

```diff
@@ -49,6 +49,8 @@ ENV OMP_NESTED="FALSE"
 
 COPY docker-entrypoint.sh /usr/local/bin/
 COPY set-num-threads.sh /usr/local/bin/
+RUN mkdir /bench
+COPY ptmsgemm.py /bench
```

Comment:
For this example bench, usually it is not appropriate to add it to the container image. Since it is only used in the k8s yaml file, many ways could put this bench file to the running pod, such as mount/configmap way. Or more flexible way to start a simple flask server, then any bench could be received and executed.